var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -256,6 +256,9 @@ def setup_dependent_environment(self, spack_env, run_env, dependent_spec):
         neuron_archdir = self.get_neuron_archdir()
         spack_env.prepend_path('PATH', join_path(neuron_archdir, 'bin'))
         spack_env.prepend_path('LD_LIBRARY_PATH', join_path(neuron_archdir, 'lib'))
+        if self.spec.satisfies('+python'):
```

Comment:
I'm a bit confused with `neuron` recipe.

Is it the proper way to make Python `neuron` dependents find it in `PYTHONPATH`?
Or should I check `+pysetup` variant here?

Reply:
@pramodk, any comments ?
I'm also not sure I understand fully the current recipe

Reply:
this looks fine to me.

@ferdonline  : how are you setting python path in python-neurodamus?

Reply:
We start neurodamus using special (also for the python version), which is linked with the python lib. As so it will always find the relative modules.
My onlky comment here would be to avoid this lib64 & lib thing. cant we properly detect and use only one?

Reply:
I have merely copy-pasted the corresponding lines from `setup_environment` to make sure their effect is propagated to dependents. :)
I have not given it much thought if it's possible to simplify it; I have seen different options for defining Python library installation folder (see also [#282](https://github.com/BlueBrain/spack/pull/272#discussion_r265008371)); and can't tell what's the "canonical" one.

Reply:
@ferdonline, just beware that if we would ever hope to let python-neurodamus take over the functionality of BGLibPy that you would have to abandon the idea to always use a 'special' as a way to start python. This is just not feasible for the ways people use BGLibPy.

Reply:
I agree @wvangeit. We will make necessary changes so that it works without special or nrniv.

@arsenius7 : Its ok if you merge this as it is now. We will take of this lib/lib64 stuff separately (across the packages).

Reply:
@ferdonline, what about Werner's question regarding `h5py~mpi`?
Would it be better to avoid `~mpi` specification for the recipe generality?